dev-docs/bidders/rubicon.md

```diff
@@ -9,7 +9,7 @@ usp_supported: true
 coppa_supported: true
 schain_supported: true
 media_types: video
-userIds: identityLink, liveIntentId, unifiedId
+userIds: identityLink, liveIntentId, sharedId, pubCommonId, pubProvidedId, unifiedId
```

Comment:
based on the pbjs PR, it looks like you guys are now supporting all user id modules, correct? if that's in fact true, this should be updated as follows:
```suggestion
userIds: britepoolId, criteo, haloId, id5Id, identityLink, intentIqId, liveIntentId, lotamePanoramaId, merkleId, netId, parrableId, pubCommonId, quantcastId, sharedId, unifiedId, zeotapIdPlusId
```